src/main/resources/odf1.3/OpenDocument-v1.3-part1-introduction_odt/styles.xml

```diff
@@ -928,13 +928,13 @@
    </text:list-level-style-number>
   </text:list-style>
   <text:list-style style:name="Appendix">
-   <text:list-level-style-number text:level="1" text:style-name="Zeichenformat" style:num-prefix="Appendix " style:num-suffix=" " style:num-format="A" style:num-letter-sync="true">
+   <text:list-level-style-number text:level="1" text:style-name="Zeichenformat" style:num-prefix="Appendix " style:num-suffix="  " style:num-format="A" style:num-letter-sync="true">
     <style:list-level-properties text:min-label-width="0.762cm" text:min-label-distance="0.127cm"/>
    </text:list-level-style-number>
-   <text:list-level-style-number text:level="2" text:style-name="Zeichenformat" style:num-suffix=" " style:num-format="1" text:display-levels="2">
+   <text:list-level-style-number text:level="2" text:style-name="Zeichenformat" style:num-prefix="Appendix " style:num-suffix="  " style:num-format="1" text:display-levels="2">
```

Comment:
i wondered what this does but apparently the prefix only applied once to the whole number so it's fine; was somehow thinking it would be applied per-level...